cli/src/main/java/org/jboss/as/cli/handlers/QuitHandler.java

```diff
@@ -41,5 +41,6 @@ public QuitHandler(String command) {
     @Override
     protected void doHandle(CommandContext ctx) {
         ctx.terminateSession();
+        System.exit(0);
```

Comment:
Calling System.exit needs to be carefully controlled or it will break embedded usage. "Quitting" an embedded CLI may not mean the user wants the embedding app to exit. On the server side we do this by using a SystemExiter interface, and all "exit" calls use that. We then swap in different impls of the interface depending on how the server is being used. The standard impl that is used when we know the server was started from the command line calls System.exit. Others do not.